master/master.cfg

```diff
@@ -261,6 +264,21 @@ test_factory.addStep(ShellCommand(
     logfiles={"console" : { "filename" : "console.log", "follow" : False }},
     decodeRC={0 : SUCCESS, 1 : FAILURE, 2 : WARNINGS, 3 : SKIPPED },
     description=["zfsstress"], descriptionDone=["zfsstress"]))
+test_factory.addStep(Git(repourl=zfs_fedora_installer_repo,
+    workdir="build/tests/zfs-fedora-installer",
+    mode="full", method="clobber", codebase="zfs-fedora-installer",
+    logEnviron=False, getDescription=True,
+    description=["cloning"], descriptionDone=["cloned"]))
+test_factory.addStep(ShellCommand(
+    workdir="build/tests/zfs-fedora-installer",
+    env={'PATH' : bin_path + ":" + zfs_path},
+    command=["runurl", bb_url + "bb-test-bootable-image.sh"],
+    haltOnFailure=False, maxTime=14400, sigtermTime=30, logEnviron=False,
+    logfiles={"console" : { "filename" : "console.log", "follow" : False },
+              "log"     : { "filename" : "output.log",  "follow" : True }},
+    decodeRC={0 : SUCCESS, 1 : FAILURE, 2 : FAILURE, 5 : FAILURE, 9 : FAILURE, 18 : SKIPPED },
+    description=["test-bootable-image"], descriptionDone=["test-bootable-image"]))
+
```

Comment:
A Fedora builder will need to be added to the `test_builders` below and we'll need to add Fedora the test slaves.  Right now we only compile for Fedora we don't running any of the tests there.  Here are the current test builders.

http://build.zfsonlinux.org/waterfall?tag=Testshttp://build.zfsonlinux.org/waterfall?tag=Tests